imjadn-v1.0-cn02.md

```diff
@@ -168,13 +168,43 @@ For complete copyright information please see the full Notices section in [Appen
 
 # 1 Introduction
 
-An Information Model (IM) defines the essential content of
-messages used in computing, independently of how those messages
-are represented (i.e., [serialized](#24-serialization)) for
-communication or storage. This Committee Note (CN) describes the
-nature of an IM, and the application of the *JSON Abstract Data
-Notation* [[JADN](#jadn-v10)] information modeling language in the
-creation and use of IMs.
+This Committee Note (CN) describes the nature of information models and the application
+of the *JSON Abstract Data Notation* [[JADN](#jadn-v10)] information modeling language
+in the creation and use of IMs.
+
+[RFC 3444](#rfc3444) discusses the difference between information models and data models:
```

Comment:
This introductory text reminds me of the comment from a NIST manager about the frequent use of number rather than titles to refer to  NIST documents (something about "the number soup"). I'd suggest something like:
> The Internet Engineering Task Force (IETF) recognized the importance of Information Models, as distinguished from Data Models, in RFC 3444: ...

That focuses a bit more on the authoritative source and the topic, vs. just a numeric document reference.